certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -111,17 +111,51 @@
                   "200": {
                       "description": "Ok",
                       "schema": {
-                          "$ref": "#/definitions/ipResult"
+                          "$ref": "#/definitions/ip"
                       }
                   }
+              },
+              "x-ms-api-annotation": {
+                "family": "IpScan",
+                "revision": 2
               }
           }
       },
-      "/api/v3/analyses/{id}": {
+      "/api/v3/ip_addresses/{ip}": {
+        "get": {
+          "summary": "Ip scan report V3",
+          "description": "Scan Ip to get full report",
+          "operationId": "VirusTotalGetIpScanV3",
```

Comment:
Deprecate this operation so is no longer available for new workflows.
https://docs.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation